CONTRIBUTING.md

```diff
@@ -0,0 +1,247 @@
+# Contributing Guidelines
+
+This document describes the existing developer tooling we have in place (and what to
```

Comment:
Sync up with @simonxiaoss I thought it is 'framework' not 'tool'

Reply:
Huh? This document is describing the _tooling_ for the LISAv3 _framework_.